tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "importsNotUsedAsValues": "error",
+    // "importsNotUsedAsValues": "error",
```

Comment:
I'm getting an` import type` TS error in `node_modules`, in `use-async-resource`, even though `skipLibCheck` is enabled. I think this is a TypeScript bug, so I've disabled the check for now.

Reply:
Hopefully `use-suspender` won't trigger this error...